components/email/src/Email.svelte

```diff
@@ -1891,23 +1884,34 @@
               </div>
               {#if _this.show_number_of_messages}
                 <span class="thread-message-count">
-                  {activeThread.messages.length
+                  {activeThread?.messages?.length > 0
                     ? activeThread.messages.length
                     : ""}
                 </span>
               {/if}
               <div class="subject-snippet-attachment">
                 <div class="subject-snippet">
-                  {#if thread?.subject}<span class="subject"
-                      >{thread?.subject}</span
-                    >{/if}
-                  <span class="snippet"
-                    >{(isDraft && activeThread?.drafts.length) ||
-                    activeThread?.messages.length
-                      ? thread.snippet.replace(/\u200C /g, "")
-                      : `Sorry, looks like this thread is currently unavailable. It may
-                    have been deleted in your provider inbox.`}</span
+                  {#if thread?.subject}
+                    <span class="subject">
+                      {thread?.subject}
+                    </span>
+                  {/if}
+                  <span
+                    class={`snippet ${
+                      activeThread.messages.length <= 0 && !isDraft
+                        ? "deleted"
+                        : ""
+                    }`}
```

Comment:
I think this would be a bit more readable
```suggestion
                    class="snippet"
                    class:deleted={activeThread.messages.length <= 0 && !isDraft}
```